src/components/Button.tsx

```diff
@@ -87,6 +87,7 @@ export type ButtonProps = Pick<
     style?: StyleProp<ViewStyle>
     hoverStyle?: StyleProp<ViewStyle>
     children: NonTextElements | ((context: ButtonContext) => NonTextElements)
+    PressableComponent?: React.ComponentType<PressableProps>
```

Comment:
Extremely neat, we can stick `PressableScale` in here now presumably?

Reply:
Yea that actually should work!